examples/app-basic/tests/appConfig.spec.ts

```diff
@@ -0,0 +1,23 @@
+import pluginJson from '../src/plugin.json';
+import { test, expect } from '@grafana/plugin-e2e';
+
+test('configuring app', async ({ page }) => {
+  await page.goto(`/plugins/${pluginJson.id}`);
+
+  const saveForm = page.getByRole('button', { name: /Save API settings/i });
+  await expect(saveForm).toBeVisible();
+
+  // reset the configured secret
+  await page.getByRole('button', { name: /reset/i }).click();
+
+  // enter some valid values
+  await page.getByLabel('API Key').fill('secret-api-key');
+  await page.getByLabel('API Url').clear();
+  await page.getByLabel('API Url').fill('http://www.my-awsome-grafana-app.com/api');
+
+  // listen for the server response on the saved form
+  const saveResponse = page.waitForResponse(`/api/plugins/${pluginJson.id}/settings`);
```

Comment:
@sunker do you know of a better way to verify a successful saved config?

Reply:
Referencing this PR: https://github.com/grafana/plugin-tools/pull/795
Will update this test when that one is available.